Runtimes/Core/core/CMakeLists.txt

```diff
@@ -210,6 +210,13 @@ add_library(swiftCore
   UTF8.swift
   UTF16.swift
   UTF32.swift
+  UTF8Span.swift
+  UTF8SpanBits.swift
+  UTF8SpanComparisons.swift
+  UTF8SpanFundamentals.swift
+  UTF8SpanInternalHelpers.swift
+  UTF8SpanIterators.swift
+  UTF8EncodingError.swift
```

Comment:
Is this sorted properly? Should they be ordered around `UTF8.swift`?

Reply:
I went through and put them in alphabetical order, thanks.